examples/v1beta1/taskruns/alpha/step-stream-support.yaml

```diff
@@ -0,0 +1,42 @@
+# This example TaskRun demonstrates how to redirect stdout and
+# stderr for Step 'digest-step'.
+#
+# The Step 'digest-step' redirects stdout to the path for the Task
+# result `digest` and stderr to the path for the Task result 'error'.
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: step-stream
+spec:
+  results:
+    - name: digest
+      description: image digest
+    - name: error
+      description: error message
+  steps:
+    - name: digest-step
+      image: gcr.io/go-containerregistry/crane
+      args:
+        - digest
+        - docker.io/library/bash
+      stdoutConfig:
+        path: $(results.digest.path)
+      stderrConfig:
+        path: $(results.err.path)
```

Comment:
I think there is a typo here 🤔 should this be `results.error.path` and not `results.err.path`?

Reply:
Yep, you are correct. I'll push an update in a few minutes.